js/app.js

```diff
@@ -79,16 +106,37 @@ articleView.handleArticleDisplay = function() {
 
 articleView.handleMainNav = function() {
     $('main-nav').on('click', '.tab', function(event) {
-      event.preventDefault();
-      $(this).parent().find('*').fadeIn();
-      $(this).hide();
+      $('.tab-content').hide();
+      $('#' + $(this).data('content')).fadeIn();
     });
-  };
-
-
 
+    $('.main-nav .tab:first').click();
+    };
+
+Article.fetchAll = function() {
+
+    $.ajax({
+      url: '/js/contents.json',
+      dataType: 'JSON',
+      type: 'GET',
+      // context: contents,
+      success: function(contents) {
+        // $(this).addClass("done");
+        Article.loadAll(contents);
+        // localStorage.rawData = JSON.stringify(contents);
+        // articleView.initIndexPage();
+        console.log('successful ajax call');
```

Comment:
@kmzurlocuva keep focusing on debugging techniques - such as this one.  They will definitely help you determine how to handle your data and if your response is coming in, as expected.